src/exchange/information/information.controller.js

```diff
@@ -35,6 +38,11 @@ angular
             this.loadPtrTooltip();
             this.loadPtrv6Tooltip();
             this.loadATooltip();
+
+            this.officeOfferService.getOfficeOfferSubscription()
+                .then((officeTenantServiceName) => {
+                    this.shouldDisplayOfficeAttached = !!officeTenantServiceName;
```

Comment:
`_(officeTenantServiceName).isString() && !_(officeTenantServiceName).isEmpty()` would be more explicit

Reply:
Wrong parameter name here, it's a boolean coming down the pipe, not a String.
Fixed by renaming .